jenkins/plugins.txt

```diff
@@ -7,6 +7,7 @@ azure-commons:1.0.5
 azure-credentials:4.0.3
 azure-keyvault:2.1
 azure-vm-agents:1.5.1
+allure:2.29.0
```

Comment:
can you give a background to this and why you want it? 

(and please add it in alphabetical order)

Reply:
is this the plugin you're after?

if so it's 
```suggestion
allure-jenkins-plugin:2.29.0
```